schema/spacewalk/spacewalk-schema-upgrade

```diff
@@ -86,6 +86,8 @@ my $my_schema_version_out = `rpm -qf --qf '%{name} %{version} %{release}\n' /etc
 if ($?) { # the directory does not exists or is not owned by any rpm
         $my_schema_version_out = `rpm -qf --qf '%{name} %{version} %{release}\n' /etc/sysconfig/rhn/oracle/main.sql`;
 }
+# Remove the leading Uyuni version, if present
+$my_schema_version_out =~ s/^([^ ]+ *)([0-9]{6,})?(\.)?([^ ]+)( *[^ ]+)$/$1$4$5/p;
```

Comment:
This change can be test at any Uyuni server by running:
```perl
my $my_schema_version_out = `rpm -qf --qf '%{name} %{version} %{release}\n' /etc/sysconfig/rhn/oracle/schema-override 2> /dev/null`;
if ($?) { # the directory does not exists or is not owned by any rpm
        $my_schema_version_out = `rpm -qf --qf '%{name} %{version} %{release}\n' /etc/sysconfig/rhn/oracle/main.sql`;
}
# Remove the leading Uyuni version, if present
$my_schema_version_out =~ s/^([^ ]+ *)([0-9]{6,})?(\.)?([^ ]+)( *[^ ]+)$/$1$4$5/p;
print $my_schema_version_out;

my $my_schema_version_out = `rpm -qf --qf '%{name} 555544.%{version} %{release}\n' /etc/sysconfig/rhn/oracle/schema-override 2> /dev/null`;
if ($?) { # the directory does not exists or is not owned by any rpm
        $my_schema_version_out = `rpm -qf --qf '%{name} 555544.%{version} %{release}\n' /etc/sysconfig/rhn/oracle/main.sql`;
}
# Remove the leading Uyuni version, if present
$my_schema_version_out =~ s/^([^ ]+ *)([0-9]{6,})?(\.)?([^ ]+)( *[^ ]+)$/$1$4$5/p;
print $my_schema_version_out;
```

It supports `YYYYMM`, but can support more digits in the future, if needed.